packages/client/src/core/board/getRandomIndex.ts

```diff
@@ -0,0 +1,5 @@
+const getRandomIndex = (upperBoard = 10) => {
+  return Math.floor(Math.random() * upperBoard)
+}
+
+export default getRandomIndex
```

Comment:
JFYI, не обязательно исправить:
```suggestion
export default (upperBoard = 10) => {
  return Math.floor(Math.random() * upperBoard)
}
```
При импорте будет браться название из имени файла